src/Components/Alert/Components/Steps/Confirmation.tsx

```diff
@@ -25,7 +25,9 @@ export const Confirmation: FC = () => {
           </Join>
         </Flex>
         <ConfirmationArtworksGridQueryRenderer
-          onClose={() => false}
+          onClose={() => {
+            dispatch({ type: "RESET" })
+          }}
```

Comment:
To ensure that the modal closes when clicking the "Show all matching artworks" button, which links to the artist page with adjusted filters as query parameters.